chains/ethereum/contract/ETH/LockRedeem_Kratos.sol

```diff
@@ -0,0 +1,271 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+contract LockRedeem {
+    //Flag to pause and unpause contract
+    bool ACTIVE = false;
+
+    // numValidators holds the total number of validators
+    uint public numValidators;
+
+    mapping(address => uint8) public validators;
+    address[] validatorList;
+
+    // Require this amount of signatures to push a proposal through,Set only in constructor
+    uint votingThreshold;
+    uint activeThreshold;
+
+    //the base cost of a transaction (21000 gas)
+    //the cost of a contract deployment (32000 gas) [Not relevant for sign trasaction]
+    //the cost for every zero byte of data or code for a transaction.
+    //the cost of every non-zero byte of data or code for a transaction.
+    uint transactionCost = 23192;
+    uint additionalGasCost = 37692;
+    uint redeemGasCharge = 10000000000000000;
+    uint validatorEarningMultiplier = 1;
+    uint rewardGas = 10000;
+
+    uint public migrationSignatures;
+    mapping(address => bool) public migrationSigners;
+    mapping(address => uint) migrationCount;
+    address [] migrationAddress;
+
+    // Default Voting power should be updated at one point
+    //int constant DEFAULT_VALIDATOR_POWER = 100;
+    uint constant MIN_VALIDATORS = 0;
+
+    //Approx
+    // 270 blocks per hour
+    // 5 blocks per min
+    // 28800 old value
+    uint256 LOCK_PERIOD;
+
+    // Variables for migrate
+    address old_contract;
+    uint signaturesrequiredformigration = 0;
+    uint migrationsignaturecount = 0;
```

Comment:
These 3 vars are extra params than existing script?

Reply:
Yes ,these are part of the the Kratos contract ,and not the Kainos one